From e61566ece12d17dbfc31016b6037e04dccc1299e Mon Sep 17 00:00:00 2001
From: Aradhya Bhatia <a-bhatia1@ti.com>
Date: Fri, 28 Feb 2025 16:05:35 +0530
Subject: [PATCH 22/25] PENDING: arm64: defconfig: Enable ILITEK ILI21X &
 ILI25X family of touch sensors

Microtips Technology USA's LVDS Panel MF-13HIEBCAF0 uses ILITEK 2511
capacitative touch sensor to provide touch features on the display.
This panel is used with TI's AM62-SK EVMs and AM62-LP-SK EVMs.

Enable ILITEK ILI21X touch sensors.

Signed-off-by: Aradhya Bhatia <a-bhatia1@ti.com>
Signed-off-by: Swamil Jain <s-jain1@ti.com>
---
 arch/arm64/configs/defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index 28665c240969..2f2344cd7bdf 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -485,6 +485,7 @@ CONFIG_INPUT_TOUCHSCREEN=y
 CONFIG_TOUCHSCREEN_ATMEL_MXT=m
 CONFIG_TOUCHSCREEN_GOODIX=m
 CONFIG_TOUCHSCREEN_GOODIX_BERLIN_SPI=m
+CONFIG_TOUCHSCREEN_ILI210X=m
 CONFIG_TOUCHSCREEN_ELAN=m
 CONFIG_TOUCHSCREEN_EDT_FT5X06=m
 CONFIG_INPUT_MISC=y
-- 
2.47.2

